dev/tests_common/pytest_plugin.py

```diff
@@ -118,6 +118,7 @@
 ).resolve()
 AIRFLOW_TESTS_DIR = AIRFLOW_SOURCES_ROOT_DIR / "tests"
 
+os.environ["AIRFLOW_TESTS_DIR"] = os.fspath(AIRFLOW_TESTS_DIR)
```

Comment:
It doesn not seem that this env variable is used here, is it ?

Reply:
I should have removed it. Removing it now